pkg/core/transaction/attribute.go

```diff
@@ -8,17 +8,21 @@ import (
 	"github.com/nspcc-dev/neo-go/pkg/io"
 )
 
+// AttrValue represents a Transaction Attribute value.
+type AttrValue interface {
```

Comment:
Fix the commit message, it's not refactor. It's "move transaction Attribute value to a designated interface".